src/video-grid/VideoTile.tsx

```diff
@@ -17,28 +17,52 @@ limitations under the License.
 import React, { forwardRef } from "react";
 import { animated } from "@react-spring/web";
 import classNames from "classnames";
+
 import styles from "./VideoTile.module.css";
 import { ReactComponent as MicMutedIcon } from "../icons/MicMuted.svg";
 import { ReactComponent as VideoMutedIcon } from "../icons/VideoMuted.svg";
 import { AudioButton, FullscreenButton } from "../button/Button";
 
-export const VideoTile = forwardRef(
+interface Props {
+  name: string;
+  showName?: boolean;
```

Comment:
I think we actually don't use this anywhere now

Reply:
Let me double check where it is used. But yea that could be right. Sorry for merging then. It felt like the merge didnt change anything needing a review 😬 my bad

Reply:
Its still used in `VideoGrid.stories.tsx`:
```tsx
<VideoGrid layout={layout} items={items}>
          {({ item, ...rest }) => (
            <VideoTile
              key={item.id}
              name={`User ${item.id}`}
              showName={items.length > 2 || item.focused}
              disableSpeakingIndicator={items.length < 3}
              {...rest}
            />
          )}
        </VideoGrid>
```
But you are right it has no effect? Should we remove it properly from everywhere?

Reply:
Yes, we should - I did it this morning in other places :p See https://github.com/vector-im/element-call/pull/514

Reply:
Okay updated the PR